test/inventory/test_terraform.py

```diff
@@ -0,0 +1,111 @@
+import terraform
```

Comment:
I've got a branch open to make complete tests for this, would you remove the tests here from the PR?